libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionTest.kt

```diff
@@ -251,13 +258,12 @@ class MapboxTripSessionTest {
     @Test
     fun getStatusImmediatelyAfterUpdateLocation() = coroutineRule.runBlockingTest {
         tripSession.start()
-        val currentDate = Date()
 
         updateLocationAndJoin()
 
-        val slot = slot<Date>()
+        val slot = slot<Long>()
         coVerify { navigator.getStatus(capture(slot)) }
-        assertTrue(slot.captured.time >= currentDate.time + navigatorPredictionMillis)
+        assertTrue(slot.captured >= TIME_NANO + navigatorPredictionMillis)
```

Comment:
nit: make this unit test be decoupled of the time setUp. 

fun getStatusImmediatelyAfterUpdateLocation() = coroutineRule.runBlockingTest {
    val startTimeNanos = TimeUnit.MICROSECONDS.toNanos(2)
    every { time.nanoTime() } returns startTimeNanos
    tripSession.start()
    
    val slot = slot<Long>()
    coVerify { navigator.getStatus(capture(slot)) }
    assertTrue(slot.captured >= startTimeNanos + navigatorPredictionMillis)
}